spotlight-client/src/VizNotes/VizNotes.tsx

```diff
@@ -25,14 +25,19 @@ import { UnknownsNote } from "./UnknownsNote";
 type VizNotesProps = {
   smallData?: boolean;
   unknowns?: Unknowns;
+  download?: () => Promise<void>;
```

Comment:
does this prop actually need to be optional?

Reply:
We do use VizNotes without `unknowns` which means also no need for `download`